wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSessionBuilder.swift

```diff
@@ -115,6 +115,7 @@ struct ZMUserSessionBuilder {
         application: any ZMApplication,
         cryptoboxMigrationManager: any CryptoboxMigrationManagerInterface,
         coreDataStack: CoreDataStack,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil,
```

Comment:
I suggest to not provide a default value.
```suggestion
        analyticsSessionConfiguration: AnalyticsSessionConfiguration?
```